### DISCUSSION TITLE
一个umi 项目 引入外部文件 react 组件 报错，求助！！

### DISCUSSION THREAD

**Discussion by yangjie21** [umijs/umi#11534] [Q&A]
<img width="1198" alt="image" src="https://github.com/umijs/umi/assets/19204745/5abba078-f63b-4a55-a46f-2b92cf69c4d8">

**Comment by fz6m** [comment]
你这个第三方组件需要依赖 context ，自己排查下需要哪个 context 吧，是第三方库没自己包还是需要自己导入 provider 包一下。
